reco_utils/common/python_utils.py

```diff
@@ -2,29 +2,50 @@
 from scipy.sparse import coo_matrix
 
 
+def exponential_decay(value, max_val, half_life):
+    """Compute decay factor for a given value based on an exponential decay
+    Args:
+        value (numeric): value to calculate decay factor
+        max_val (numeric): value at which decay factor will be 1
+        half_life (numeric): value at which decay factor will be 0.5
+    Returns:
+        (float): decay factor
```

Comment:
it occurs to me that if max_val < max(x) this will actually exponentially <i>increase</i> the value which doesn't seem like the desired behavior, should those values be left as is or set as zero?